COPYRIGHT

```diff
@@ -1,4 +1,4 @@
-Copyright © 2014 Pulp Project developers.
+Copyright © 2017 Pulp Project developers.
```

Comment:
Would it be best to leave this at 2014?  I don't think it matters one way or the other in this case

Reply:
I think this should be left as 2014: 

> The general rule is that the year to include in a copyright notice is the year of first publication of the work. [0]

[0] http://www.copyrightlaws.com/copyright-notice-year/